nixos/modules/services/audio/spotifyd.nix

```diff
@@ -78,6 +79,10 @@ in
       }
     ];
 
+    warnings = optionals (cachePath != defaultCachePath) [
+      ''Changing the `cache_path` setting will not ensure that the service will have read/write permissions for that directory.''
+    ];
```

Comment:
I don't like that this warning will be displayed forever when the option is changed. IMO we should move this into the options description.